web/src/compose.js

```diff
@@ -75,7 +75,7 @@ export function compute_show_video_chat_button() {
 
 export function update_video_chat_button_display() {
     const show_video_chat_button = compute_show_video_chat_button();
-    $(".compose_bottom_top_container .video_link").toggle(show_video_chat_button);
+    $(".compose_buttons_container .video_link").toggle(show_video_chat_button);
```

Comment:
These renames are all great. Let's follow [Zulip's new preferred style](https://zulip.readthedocs.io/en/latest/subsystems/html-css.html#use-clear-unique-names-for-classes-and-object-ids) and use hyphens (_kebab case_) rather than underscores to separate the words.